test/runtests.jl

```diff
@@ -44,6 +44,7 @@ test_pattern_get_surface()
 pth = joinpath(dirname(dirname(@__FILE__)), "samples")
 fls = filter(str->endswith(str,".jl"), readdir(pth))
 for fl in fls
+    println(fl)
```

Comment:
I did choose the IOBuffer as interface, libcairo actually only needs a function call https://www.cairographics.org/manual/cairo-PNG-Support.html#cairo-image-surface-create-from-png-stream that copies sequentially from input to an internal buffer. As my initial target was "read png from string" IOBuffer seemed a good match.